input/fsh/SD_Radiotherapy.fsh

```diff
@@ -175,12 +175,8 @@ Description: "A volume of the body used in radiotherapy planning or treatment de
 * location from RadiotherapyTreatmentLocationVS (required)
 * location ^short = "Body Location Code."
 * location ^definition = "A code and qualifiers (via extensions) specifying the TG263 body structure comprising the irradiated volume."
-* location.extension contains
-     BodyLocationQualifier named locationQualifier 0..*   and
-     LateralityQualifier named lateralityQualifier 0..1
-* locationQualifier 0..0
-* location.extension[locationQualifier].value[x] from RadiotherapyTreatmentLocationQualifierVS (required)
-* identifier and location and description and patient and morphology and location.extension[locationQualifier] and location.extension[lateralityQualifier] MS
+* locationQualifier from RadiotherapyTreatmentLocationQualifierVS
```

Comment:
There are two places where laterality comes into play:
   - Condition.bodySite
        - this field had **no notion of qualifiers**, so we added two extensions -- locationqualifier and lateraity qualifier.   The laterality was separate from the general location qualifier.
   - BodyStructure.locationQualifier
      - since bodystructure had a locationQualifier field, the decision is whether to:
           1) make it 0..0 and add two qualifiers (a la COndition.bodySite).  We did this first.   It is unclear whether this violates HL7 FHIR etiquette.    We decided it was kind of shady, and backed away from this solution.
           2) request a change in R5 -- I did this, and you successfully litigated the lawsuit.  Doesn't help us now.
           3) Conform to the current BodyStructure, and include the laterality qualifiers in the VS for locationQualifier.
                 - This is what I was trying to do, but I effed up the VS by failing to include the left/right/left+right/midline values in the VS...This is corrected.
            4) We didn't consider, but...could add a laterality extension onto the locationQualifier field. 
            
 So far, we've done #1, then #2 + #3.   
 Would you prefer that we do #2 + #4?
 
 The QA issues that I failed to detect now pass, after repairing the VS to align with the BodyStructure resource structure.